posthog/api/app_metrics.py

```diff
@@ -27,28 +32,24 @@ class AppMetricsViewSet(TeamAndOrgViewSetMixin, mixins.RetrieveModelMixin, views
 
     def retrieve(self, request: request.Request, *args: Any, **kwargs: Any) -> response.Response:
         try:
-            # probe if we have a valid uuid, and thus are requesting metrics for a batch export
-            uuid.UUID(kwargs["pk"])
+            rows = self.get_batch_export_runs_app_metrics_queryset(batch_export_id=kwargs["pk"])
+
+            dates = [row["dates"].strftime("%Y-%m-%d") for row in rows]
+            successes = [row["successes"] for row in rows]
+            failures = [row["failures"] for row in rows]
             return response.Response(
                 {
-                    "metrics": [
-                        {
```

Comment:
Not sure why this was a list, I made my thing match the schema returned by plugin metrics (which doesn't return a nested list). However, if we need a list, we can nest this again.